.github/workflows/ci.yml

```diff
@@ -76,12 +76,13 @@ jobs:
           go-version: 1.17.x
       - name: make test
         run: make test
-  e2e:
-    name: e2e test
+  kubernetes:
+    name: Kubernetes
     needs: build
     runs-on: ubuntu-18.04
-    env:
-      ARTIFACTS_PATH: ${{ github.workspace }}/karmada-e2e-logs/
+    strategy:
+      matrix:
+        k8s: [v1.21.10, v1.22.7, v1.23.4]
```

Comment:
Should we always use the latest patch release?

Reply:
Personally, I don't think this is necessary. As Kind keeps building new image versions of k8s, updates will be fast. For us, we need to make sure that the Kubernetes minor version number is feasible.